website/docs/terms/dml.md

```diff
@@ -0,0 +1,103 @@
+
+
+## Types of DML Statements
+
+The primary DML statements are `SELECT`, `INSERT`, `DELETE`, and `UPDATE`. With the exception of `SELECT` statements, all of the others are only applicable to data within tables in a database. The primary difference between `SELECT` and all the other DML statements is its impact to row-level data:
+
+- To *change* the actual data that lives in tables, use `INSERT`, `DELETE`, and `UPDATE` statements
+- To *access* the data in databse object, use `SELECT` statements
+
+:::important Important
+For the most part, the syntax for DML statements are pretty universal across [most databases that dbt supports](https://docs.getdbt.com/docs/available-adapters) including Google Bigquery, Databricks, Postgres, Amazon Redshift, and Snowflake. Regardless, please use the “Further Reading” section to see the specifics on how the following DML statements would be implemented in your database of interest!
+:::
+
+### SELECT
+
+Ah, our favorite of DML statements! This is the SQL we all know and love (most of the time). Because the `SELECT` statement allows you to access and manipulate data that exists in database objects, it makes it the true powerhouse in data analysis and analytics engineering. 
+
+You write `SELECT` statements to create queries that build data models and perform robust analysis. With `SELECT` statements, you can join different views and tables, qualify data by setting filters, apply functions and operators on the data, and more. `SELECT` statements, unlike `INSERT`, `DELETE`, and `UPDATE`, don’t actually change the row-level value stored in the tables/views. Instead, you write `SELECT` statements to express the business logic needed to perform analysis.
+
+All `SELECT` statements need three elements: a `SELECT` clause in the beginning, the actual field selection and manipulation, and a `FROM` statement which is specifying which database object you’re trying to access.
+
+Here’s an example `SELECT` statement:
+
+```sql
+select
+
+	payment_method,
+	sum(amount) AS amount
+
+from {{ ref(‘raw_payments’) }}
+group by 1
+```
+
+In this example, your selection of the `payment_method` column and summation of the `amount` column is the meat of your query. The `from {{ ref(‘raw_payments’) }}` specifies the actual table you want to do the selecting from.
+
+### INSERT
+
+Using the `INSERT` DML command, you can add rows to a table that exists in your database. To be honest, data folks are rarely inserting data into tables manually with the `INSERT` command. Instead, data team members will most often use data that’s already been inserted by an <Term id="elt" /> tool or other data ingestion process.
+
+You can insert a record for myself [in jaffle_shop’s](https://github.com/dbt-labs/jaffle_shop) `raw_customers` table like this:
+
+```sql
+INSERT INTO raw_customers VALUES (101, ‘Kira’, ‘F.’);
```

Comment:
Kira and F should be actual quote-enclosed strings (they'll turn into green strings)

Reply:
Fixed!